.circleci/continue-config.yml

```diff
@@ -81,6 +78,9 @@ jobs:
           pkg-manager: yarn
       - run: yarn cypress install
       - run: yarn cypress verify
+      - run:
+          name: 'echo REACT_APP_ETH_MAINNET_API_KEY from CI context'
+          command: echo $REACT_APP_ETH_MAINNET_API_KEY
```

Comment:
all these are just for debugging and can now be removed, right? @sshmm

Reply:
@crabsinger  as far as I know, it's necessary to echo the variables to make them visible to the rest of the scripts.

Reply:
@sshmm hmm, im very surprised by that. have you tried removing this echo?

Reply:
> @sshmm hmm, im very surprised by that. have you tried removing this echo?

@crabsinger  you are right. it's not required.

Reply:
@crabsinger  I have removed echo commands and rebased the branch